packages/transmute-cli/src/commands/provision.js

```diff
@@ -51,47 +57,41 @@ export function aks( dryrun, myResourceGroup, myAKSCluster, myNodeCount, myNodeS
   if (GenSSHKeys) {
     gensshkeys_opt = ' --generate-ssh-keys';
   }
-  if (myAKSCluster == undefined ) {
+  if (myAKSCluster == undefined) {
     throw 'You need to define a clustername';
-  }
-  else {
+  } else {
     var akscluster_opt = ' --name ' + myAKSCluster;
   }
-  if (myNodeCount == undefined ) {
+  if (myNodeCount == undefined) {
     throw 'You need to define a number of nodes';
-  }
-  else {
+  } else {
     var nodes_opt = ' --node-count ' + myNodeCount;
   }
-  if (myNodeSize == undefined ) {
-    console.log('no size given using default')
+  if (myNodeSize == undefined) {
+    console.log('no size given using default');
     var nodesize_opt = ' ';
-  }
-  else {
+  } else {
     var nodesize_opt = ' --node-vm-size ' + myNodeSize;
   }
-  if (myResourceGroup == undefined ) {
+  if (myResourceGroup == undefined) {
     throw 'You need to define a group';
-  }
-  else {
+  } else {
     var group_opt = ' --resource-group ' + myResourceGroup;
   }
-  var prov_cmd = 'az aks create '
-    + group_opt
-    + nodes_opt
-    + nodesize_opt
-    + akscluster_opt
-    + gensshkeys_opt;
+  var prov_cmd =
+    'az aks create ' +
+    group_opt +
+    nodes_opt +
+    nodesize_opt +
+    akscluster_opt +
+    gensshkeys_opt;
   run.ner(prov_cmd);
   console.log(prov_cmd);
-  var prov_cmd = 'az aks get-credentials '
-    + akscluster_opt
-    + group_opt
-  console.log( prov_cmd );
-  if (dryrun == 'true' ) {
-    console.log( '<--dry run-->' );
-  }
-  else {
-    run.ner( prov_cmd );
+  var prov_cmd = 'az aks get-credentials ' + akscluster_opt + group_opt;
+  console.log(prov_cmd);
+  if (dryrun == 'true') {
+    console.log('<--dry run-->');
```

Comment:
console.debug / console.info

Reply:
The real solution here is to use a propper logger like winston:

https://stackoverflow.com/questions/25532778/node-js-console-log-vs-console-info

For example:

https://github.com/TrueBitFoundation/truebit-os/blob/master/os/logger.js